concent_api/utils/logging.py

```diff
@@ -1,112 +1,121 @@
-from logging                        import getLogger
+from logging import getLogger
 
-from golem_messages.message.base    import Message
-
-from utils.helpers                  import get_field_from_message
+from golem_messages.message.base import Message
 
+from utils.helpers import get_field_from_message
 
 logger = getLogger(__name__)
 
 
-def log_message_received(message: Message, client_public_key: str):
-    task_id = get_task_id_for_logging(message)
-    subtask_id = get_subtask_id_for_logging(message)
-    logger.info('A message has been received in `send/` -- MESSAGE_TYPE: {} -- TASK_ID: {} -- SUBTASK_ID: {} -- CLIENT PUBLIC KEY: {}'.format(
-        type(message).__name__,
-        task_id,
-        subtask_id,
-        client_public_key,
-    ))
-    logger.debug('A message has been received in `send/` -- MESSAGE: {}'.format(
-        message,
-    ))
+def replace_key_to_unavailable_instead_of_none(log_function):
+    def wrap(*args):
+        if args[1] is None:
+            args_list = list(args)
+            args_list[1] = 'UNAVAILABLE'
+            args = log_function(*tuple(args_list))
```

Comment:
`log_function` will not be called when `if args[1] is None:` condition is not met.